rest_server/config.py

```diff
@@ -0,0 +1,59 @@
+"""Config settings."""
+
+
+import logging
+from urllib.parse import quote_plus
+
+# local imports
+from rest_tools.server.config import from_environment  # type: ignore[import]
+
+# --------------------------------------------------------------------------------------
+# Get constants from environment variables
+
+
+_config_env = from_environment(
```

Comment:
I dislike calling this on import, as if there are any hard requirements you can't even get the help message without setting them (see LTA and https://github.com/WIPACrepo/lta/pull/136).

Reply:
okay, check out 2d939d1